web/src/components/Prices/HollaexToken.js

```diff
@@ -0,0 +1,570 @@
+import React, { useEffect, useState } from 'react';
```

Comment:
Some texts are not directly editable, also better to define some of them as formatted strings.

Reply:
I've updated the string format in possible places.